aas_core_meta/v3.py

```diff
@@ -5544,22 +5611,27 @@ class Data_specification_IEC_61360(Data_specification_content):
     Data Type
     """
 
-    definition: Optional[List["Lang_string_text_type"]]
+    definition: Optional[List["Lang_string_definition_type_iec_61360"]]
     """
     Definition in different languages
     """
 
     value_format: Optional[Non_empty_XML_serializable_string]
     """
     Value Format
+    
+    .. note::
+    
+        The value format is based on ISO 13584-42 and IEC 61360-2.
+        
     """
 
     value_list: Optional["Value_list"]
     """
     List of allowed values
     """
 
-    value: Optional[str]
+    value: Optional["Lang_string_short_name_type_iec_61360"]
```

Comment:
Why is the name of the type "Lang_string_short_name_type_iec_61360" and not just "Short_name_type_iec_61360" as specified? Similar for other types starting with Lang_string